packages/support/src/Commands/Concerns/CanReadModelSchemas.php

```diff
@@ -107,6 +96,61 @@ protected function guessBelongsToRelationshipTitleColumnName(AbstractAsset $colu
             return 'title';
         }
 
-        return $schema->getPrimaryKey()->getColumns()[0];
+        return collect($schema->getIndexes($table))->firstWhere('primary')['columns'][0] ?? 'id';
+    }
+
+    protected function parseColumnType(array $column): array
+    {
+        $type = match ($column['type']) {
+            'tinyint(1)', 'bit' => 'boolean',
+            'varchar(max)', 'nvarchar(max)' => 'text',
+            default => null,
+        };
+
+        $type ??= match ($column['type_name']) {
+            'boolean', 'bool' => 'boolean',
+            'char', 'bpchar', 'nchar' => 'char',
+            'varchar', 'nvarchar' => 'string',
+            'integer', 'int', 'int4', 'tinyint', 'smallint', 'int2', 'mediumint', 'bigint', 'int8' => 'integer',
+            'decimal', 'numeric' => 'decimal',
+            'float', 'real', 'float4' => 'float',
+            'double', 'float8' => 'double',
+            'money', 'smallmoney' => 'money',
+            'date' => 'date',
+            'time', 'timetz' => 'time',
+            'datetime', 'datetime2', 'smalldatetime', 'datetimeoffset' => 'datetime',
+            'timestamp', 'timestamptz' => 'timestamp',
+            'text', 'tinytext', 'longtext', 'mediumtext', 'ntext' => 'text',
+            'json', 'jsonb' => 'json',
+
+            // 'binary', 'varbinary', 'bytea', 'image', 'blob', 'tinyblob', 'mediumblob', 'longblob' => 'binary',
+            // 'uuid', 'uniqueidentifier' => 'uuid',
+            // 'enum' => 'enum',
+            // 'set' => 'set',
+            // 'inet', 'cidr', 'macaddr', 'macaddr8' => 'string',
+            // 'bit', 'varbit' => 'bit',
+            // 'xml' => 'xml',
+            // 'year' => 'year',
+            // 'interval' => 'interval',
+            // 'geometry', 'geometrycollection', 'linestring', 'multilinestring', 'multipoint', 'multipolygon', 'point', 'polygon', 'box', 'circle', 'line', 'lseg', 'path' => 'geometry',
+            // 'geography' => 'geography',
+            // 'tsvector', 'tsquery' => 'text',
```

Comment:
These column types are also possible, I put them here for you to do the mapping as you want. For example you may want to have a dropdown for `enum` column type.